pandas/core/ops.py

```diff
@@ -436,27 +436,33 @@ def _get_op_name(op, special):
     'eq': {'op': '==',
            'desc': 'Equal to',
            'reverse': None,
-           'df_examples': None},
+           'df_examples': None,
+           'others': None},
```

Comment:
If we don't use `reverse`, `df_examples` and `others`, can we get rid of them instead of having all them to `None`?

Reply:
`df_examples` was kept from original docs and not removed for code consistency if used elsewhere. Can remove and test docstrings.

Reply:
`reverse` is needed for compatibility with arithmetic operators as they are included in same dictionary, `_op_descriptions`